controllers/feeds/getFeedsV2.js

```diff
@@ -17,13 +17,14 @@ const {
   modifyAnonimityPost,
   isPostBlocked,
   modifyReactionsPost,
-  modifyFeedIsFollowingTarget
+  isUserFollowAuthor
 } = require('../../utils/post');
 const {DomainPage, Locations, User, UserFollowUser} = require('../../databases/models');
 const RedisDomainHelper = require('../../services/redis/helper/RedisDomainHelper');
 const {ACTIVITY_THRESHOLD} = require('../../config/constant');
 const UsersFunction = require('../../databases/functions/users');
 const UserFollowUserFunction = require('../../databases/functions/userFollowUser');
+const roundingKarmaScore = require('../../helpers/roundingKarmaScore');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `isUserFollowAuthor` and `roundingKarmaScore` functions are new additions. Ensure that these functions are correctly implemented and tested. Also, make sure they are exported from their respective modules.

```diff
+  isUserFollowAuthor
 } = require('../../utils/post');
 const {DomainPage, Locations, User, UserFollowUser} = require('../../databases/models');
 const RedisDomainHelper = require('../../services/redis/helper/RedisDomainHelper');
 const {ACTIVITY_THRESHOLD} = require('../../config/constant');
 const UsersFunction = require('../../databases/functions/users');
 const UserFollowUserFunction = require('../../databases/functions/userFollowUser');
+ const roundingKarmaScore = require('../../helpers/roundingKarmaScore');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->